src/components/UpdateAppModal/BaseUpdateAppModal.js

```diff
@@ -33,31 +35,42 @@ class BaseUpdateAppModal extends PureComponent {
                     onSubmit={this.submitAndClose}
                     onClose={() => this.setState({isModalOpen: false})}
                     isVisible={this.state.isModalOpen}
+                    type={this.props.isSmallScreenWidth
+                        ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED : CONST.MODAL.MODAL_TYPE.CONFIRM}
                 >
-                    <HeaderWithCloseButton
-                        title="Update App"
-                        onCloseButtonPress={() => this.setState({isModalOpen: false})}
-                    />
-                    <Text style={[styles.textLabel, styles.p4]}>
-                        A new version of Expensify.cash is available.
-                        Update now or restart the app at a later time to download the latest changes.
-                    </Text>
-                    {this.props.onSubmit && (
+                    <View style={[styles.m5]}>
+                        <View style={[styles.flexRow]}>
```

Comment:
```suggestion
                        <View style={styles.flexRow}>
```

Reply:
Why does the header need to be wrapped in a view w/ `flexRow` ?

Reply:
Because <Header> component does have another view and it has `flex1`.
In other places <Header> component is used inside a <view> with `flexRow`. 
If i remove `flexRow` from here it won't show in the mobile app.

Reply:
Ok, thanks for explaining.